client/src/bundles/UserForms.js

```diff
@@ -1,9 +1,4 @@
-/**
- * @file Manages the multi-step navigation.
- */
-
-import jQuery from 'jquery';
-import i18n from 'i18n';
+/* global i18n, jQuery */
```

Comment:
This only used to work because the webpack externals config was being added - but that's objectively the wrong way to handle this, since our config for webpack externals is for the CMS and also includes a lot of things that _won't_ be exposed in the frontend.